Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
```

Comment:
I'm not sure why this works for you.

`Run()` would run the game and not return until Exit() is called.  `RunOneFrame()` is supposed to run one fame and return.

So if `RunOneFrame()` isn't working for you we need to fix it.  We shouldn't switch things to `Run()`.